types/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// TypeScript Version: 3.5
+
+import {Node, Parent} from 'unist'
+
+declare namespace unistUtilModifyChildren {
+  type Modifier = (
+    node: Node,
+    index: number,
+    parent: Parent
+  ) => number | void
+
+  type Modify = (tree: Node) => undefined
```

Comment:
out of curiosity, what is the difference between the no return case from `Modifier` which is typed as `void` and the no return case of `Modify` which is typed as `undefined`? Is that difference intentional?

Reply:
The `Modifier` you pass can return a number or have no return statement. Wheres [this](https://github.com/syntax-tree/unist-util-modify-children/blob/master/index.js#L24) returns `undefined` (if I‘m correct).

Reply:
I think it may be `void` as well
https://github.com/wooorm/array-iterate/blob/1e0468b08d00e3d4684edf33bbe4d3a981109f06/index.js
does not have a return statement.

Reply:
Alternatively I can remove the return there and use `void` here.

Reply:
I'd lean slightly towards using `void` here.
More important in this context is that they types are consistent within the package (both `undefined` or both `void`)